staging/src/k8s.io/apiserver/pkg/server/options/events.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+// ProcessInfo holds the apiserver process information used to send events
+type ProcessInfo struct {
+	// Name of the api process to identify events
+	Name string
+
+	// Namespace of the api process to send events
+	Namespace string
+}
+
+// DefaultProcessInfo returns the default process information for a generic api server
+func DefaultProcessInfo() *ProcessInfo {
+	return &ProcessInfo{
+		Name:      "kube-apiserver",
```

Comment:
a default of `kube-apiserver` doesn't belong in the generic k8s.io/apiserver component. @deads2k, is there already a name or identity associated with a generic apiserver process?